### ISSUE TITLE
Links to your Sketchfab account (for photogrammetry and 3D modelling)

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2021-2022#25]
Please share a link to your Sketchfab account in this thread, so that we can have access to your photogrammetry exercise and 3D models. (You can delete any items you don't want to keep after the end of the semester.)

**Comment by Ghilaevansky** [comment]
Here is my design. I tried my best. Personally, as a non techy person I found it really complicated to use this software. 

![Screen Shot Sketchfab Ghila](https://user-images.githubusercontent.com/97957756/153723362-607d4c79-564b-4929-bd03-2076593adb76.png)

Unfortunately, my work didn't save at the end, but this is a screenshot of what I had early on.

**Comment by lettychardon** [comment]
I am also attaching a screenshot as I am struggling with SketchFab. I also couldn't move the images away from the walls so it probably would not have worked anyway! I had also built the roof and was doing the pediments when the software crashed and closed so I lost some progress... Might try again tomorrow but this is what I have for now:

![Screenshot (22)](https://user-images.githubusercontent.com/92744392/153768677-35d639cc-821c-4c0f-95a5-07a381e301e4.png)

**Comment by katie-goodman** [comment]
Here is my attempt at the 3D modelling exercise: https://skfb.ly/osCuQ

I had a bit of difficulty cutting back the cornice, but I otherwise really enjoyed the process! I have also attached a screenshot of my finished model, as I couldn’t figure out how to get the textured walls to appear once uploaded to Sketchfab.

<img width="1440" alt="Screenshot 2022-02-14 at 12 41 12" src="https://user-images.githubusercontent.com/92084220/153874851-ace122ca-b21e-49aa-b601-e0feafe97d24.png">

**Comment by gabrielbodard** [comment]
Yes, textures sometimes disappear in Sketchfab depending on what format you export as and how you upload. I don't remember the details, but we should experiment…

(There are a few examples previously uploaded at https://sketchfab.com/search?q=ekklesiasterion&type=models , with varying success, as you can see! About five of them look like they were based on this exercise…)

**Comment by molmay** [comment]
Here is my attempt! I didn't understand how to make the external offset of the pediment follow the cornice down so it currently just looks like two beams on top of the roof! 
![Screenshot 2022-02-14 at 3 28 14 pm](https://user-images.githubusercontent.com/97958069/153893416-990c1dd6-fff6-4735-90dd-62fcca261946.png)

**Comment by Dasha-Barkova** [comment]
![Снимок экрана 2022-02-15 в 1 06 55](https://user-images.githubusercontent.com/75930482/153973365-fd62e4ec-7156-48ce-9743-f3de4db139da.png)
![Снимок экрана 2022-02-15 в 1 06 40](https://user-images.githubusercontent.com/75930482/153973370-9aab5cb7-f40c-455e-8da0-774b8dc78da6.png)

I liked this exercise yet I had some difficulties with scaling the model. I also couldn't figure out how to change the language from Russian to English hence why it took me longer than I expected. I want to continue working on this model in SketchFab and hopefully, I can finish it sometime soon! :)

**Comment by ellie919** [comment]
<img width="444" alt="Screenshot 2022-02-15 at 01 22 06" src="https://user-images.githubusercontent.com/98793183/153974732-663e4a56-f791-49fe-9aac-9ebf997ffb97.png">
I also had issues with the cornices and adding texture to the walls

**Comment by lewis-rhiannon** [comment]
For a first go sketch up wasn't too bad to navigate, with the PDF for guidance and jumping in and out of youtube tutorials... But I think that I got the original width of the building wrong, which threw out a few of the stages down the line. 

I had problems with importing textures, I would import them and they looked fine and then I would make another edit to the building and they would disappear... (I'm pretty sure they're not in the right place to match the PDF instructions on my last attempt). I wasn't able to import the Venus de Milo, but I was able to import a statue of Athena. 

With all these improvisations, and wrong turns, I've ended up with something that sort of resembles what we were set but wouldn't be a viable model for research (it's also not finished yet). This did made me appreciate the level of familiarity you have would have to have to have with a source and modelling tool to be able to produce a scholarly model.

![Screenshot 2022-02-15 at 10 32 05](https://user-images.githubusercontent.com/62398935/154046416-fcaadf24-d192-484e-9539-fd08c78070dc.png)
